clientapi/routing/register.go

```diff
@@ -791,7 +841,11 @@ func completeRegistration(
 	}
 
 	// Increment prometheus counter for created users
-	amtRegUsers.Inc()
+	if isGuest {
+		amtRegUsers.Inc()
```

Comment:
You mixed up the counters here (you're increasing `amtRegUsers` if we're registering a guest user and `amtGuestUsers` if we're registering a normal user).